### PR TITLE
azure pipeline

### PR DESCRIPTION
**Pull request by srwopschall**
Adding azure pipeline file.

### CODE REVIEW

File: host-configs/docker/clang@10.0.0.cmake [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+# CALIPER not built
+
+# PETSC not built
+
+set(RAJA_DIR "${TPL_ROOT}/raja-2022.03.0-gvquqeduwans5kcq5ngysvjwrx3q7jrq" CACHE PATH "")
+
+set(SUNDIALS_DIR "${TPL_ROOT}/sundials-6.4.1-gzb3udgkn74t62ropfu5kuwz275ezkh2" CACHE PATH "")
+
+set(UMPIRE_DIR "${TPL_ROOT}/umpire-2022.03.1-5u3fkkj6euykttrc5usvncfcnhbhofud" CACHE PATH "")
```
Comment (ebchin) [inline host-configs/docker/clang@10.0.0.cmake:96]: Do these TPLs need to be updated to match the ones Tribol uses?

File: host-configs/docker/clang@10.0.0.cmake [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+# Code checks disabled due to disabled devtools
+
+set(SERAC_ENABLE_CODE_CHECKS OFF CACHE BOOL "")
+
+set(ENABLE_CLANGFORMAT OFF CACHE BOOL "")
+
+set(ENABLE_CLANGTIDY OFF CACHE BOOL "")
+
+set(ENABLE_DOCS OFF CACHE BOOL "")
+
```
Comment (ebchin) [inline host-configs/docker/clang@10.0.0.cmake:111]: Same comment as above; should these match the tribol devtools entries?

**Review by ebchin — APPROVED** [review]
Thanks @white238 !

File: host-configs/docker/clang@10.0.0.cmake (hunk 1 above)
Comment (white238) [reply host-configs/docker/clang@10.0.0.cmake:96]: These match what are already built and existing in the docker container from Serac.

**Pull request merged**